Automattic/wp-calypso
# Add RUM data to indicate if the user was bootstrapped

#### Changes proposed in this Pull Request

* Adds a new field `bootstrapped` to the user object to indicate it was bootstrapped by the server.
* Reports the above field to RUM stats

#### Testing instructions

* Checkout this branch and start the project with `yarn start`. Open the Network tab in the dev tools and navigate to <http://calypso.localhost:3000/read>. Locate a request to `/logstash` and inspect the payload, it should contain `userBootstrapped:false`

* Enable user boostrapping (PCYsg-5YE-p2) and repeat the process above. You should see `userBootstrapped:true` now.
